samples/getting_started/Superposition.qs

```diff
@@ -1,22 +1,22 @@
-/// # Sample
-/// Superposition
+/// # Summary
+/// Superposition sample
 ///
 /// # Description
 /// This Q# program sets a qubit in a superposition of the computational basis
 /// states |0〉 and |1〉 by applying a Hadamard transformation.
 operation Main() : Result {
+    // Allocate a qubit. Qubit is in |0〉 state after allocation.
+    use qubit = Qubit();
     // Qubits are only accessible for the duration of the scope where they
     // are allocated and are automatically released at the end of the scope.
-    use qubit = Qubit();
 
     // Set the qubit in superposition by applying a Hadamard transformation.
     H(qubit);
 
-    // Measure the qubit. There is a 50% probability of measuring either
-    // `Zero` or `One`.
-    let result = M(qubit);
+    // Show the quantum state (when running on a simulator).
+    Std.Diagnostics.DumpMachine();
```

Comment:
It looks like the only difference between this sample and the "RandomBit.qs" sample is the presence of this `DumpMachine`. Is there a reasonable way we can combine those? My first instinct is to drop RandomBit.qs since now there is a RandomBits sample that produces 5 bits and covers the same concepts, and leave this sample as a basic one for the concept of superposition.

Reply:
Probably a good idea to drop RandomBit.qs altogether. I liked having a separate stand-alone sample for a random bit, but after I changed RandomBits to include a dedicated function GenerateRandomBit there's not much need for a separate Random bit...
I removed it.